minidec/main.rs

```diff
@@ -59,6 +67,7 @@ fn main() {
         //println!("Local Variable info: {:?}", rfn.locals);
         let mut writer: IRWriter = Default::default();
         writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ff);
+        writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ffm);
```

Comment:
I don't know, why it says "All checks have failed". For me it compiles and all tests pass.